api/build.gradle

```diff
@@ -250,7 +250,8 @@ spotless {
 
 test {
     useJUnitPlatform()
-    maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
+    maxParallelForks = System.getenv('CIRCLECI') == 'true' ? 1 :
```

Comment:
looks good - could you add a brief comment explaining why we're doing it this way?

Reply:
Good call.